config/install/search_api.index.event_invites_and_chat_appointments.yml

```diff
@@ -0,0 +1,181 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.dip_chat_appointment.field_appointmentdate
+    - field.storage.dip_chat_appointment.field_counselor
+    - field.storage.dip_chat_appointment.field_moderators
+    - field.storage.living_spaces_event.field_end_date
+    - field.storage.living_spaces_event.field_start_date
+    - search_api.server.living_spaces
+  module:
+    - search_api_solr
+    - dip_chat_counseling
```

Comment:
This index depends on module that is not part of profile.
I think we should add this index to dip_chat_counseling module